db/python/tables/sequencing_group.py

```diff
@@ -343,7 +343,7 @@ async def get_samples_create_date_from_sgs(
         _query = """
         SELECT sg.id, min(s.row_start)
         FROM sequencing_group sg
-        INNER JOIN sample s ON s.id = sg.sample_id
+        INNER JOIN sample FOR SYSTEM_TIME ALL s ON s.id = sg.sample_id
```

Comment:
Hey @illusional I am still getting 'AssertionError: No ratio found for date 2022-01-19' for SAMPLE_CREATE_DATE after updating DB with the correct time_completed.
I reckon this is the corporate, can't really test unless we deploy. Without 'FOR SYSTEM_TIME ALL' it only return last updated timestamp and not create_date as function name suggest.

Reply:
Oh yep, sounds good, thanks @milo-hyben!
Let's give deploying a crack. 

I think this is probably the true fix.